packages/teleport-component-generator-html/src/plain-html-mapping.ts

```diff
@@ -16,10 +16,11 @@ export const PlainHTMLMapping: Mapping = {
       dependency: {
         type: 'package',
         path: 'dangerous-html',
-        version: '0.1.12',
+        version: '1.0.0',
         meta: {
           importJustPath: true,
-          importAlias: 'https://unpkg.com/dangerous-html@0.1.12/dist/default/lib.umd.js',
+          importAlias:
+            'https://unpkg.com/@teleporthq/thq-lit-components@1.0.0/dist/thq-lit-components.umd.js',
```

Comment:
And we need to improve this, or else if we are using just `dangerous-html`. It installs the `date-time-primitive` too even if we are not using. Since these are `umd` builds. The bundlers can't tree shake them